crates/engine/src/lib.rs

```diff
@@ -330,6 +336,18 @@ impl<T: Default> ExecutionContext<T> {
                 wasi_ctx.preopened_dir(Dir::open_ambient_dir(host, ambient_authority())?, guest)?;
         }
 
+        if component.core.wasm.enable_monitoring {
+            let spin_dir = match dirs::home_dir() {
+                Some(h) => h.join(SPIN_HOME),
+                None => env::current_dir()?,
+            };
+
+            wasi_ctx = wasi_ctx.preopened_dir(
+                Dir::open_ambient_dir(&spin_dir, ambient_authority())?,
+                Path::new("/.spin/"),
```

Comment:
How should this be configured?

Reply:
I think initially it doesn't need to be.